api/nakadi-event-bus-api.yaml

```diff
@@ -890,6 +1149,32 @@ definitions:
           available event in the partition.
         type: string
 
+    StreamInfo:
+      type: object
+      description: |
+        This object contains general information about the stream. Used only for debugging
+        purposes. We recommend logging this object in order to solve connection issues. Clients
+        should not parse this structure.
+
+    TooManyRequests:
```

Comment:
Just to clear on the options. If you don't want to commit now to tracking the number of requests in a window, the `X-` headers can be dropped and just use the `Retry-After` header. The `X-` headers can be added later as an extension.